test/hocr-check/test-hocr-check.tsht

```diff
@@ -2,5 +2,7 @@
 
 plan 2
 
-file_exists sample.hocr
-exec_ok hocr-check sample.hocr
+TESTDATA="../legacy"
+
+exec_ok hocr-check "$TESTDATA/sample.html"
+cat "$TESTDATA/sample.html" | hocr-check && ok "Check from stdin"
```

Comment:
Why are using the `legacy/` directory for storing the files instead the directory `hocr-check/` directly? The name does no sound correct and you mention in a previous commit to do it the other way: https://github.com/tmbdev/hocr-tools/pull/10#discussion_r63781985